src/Microsoft.Extensions.DependencyModel/DependencyContextPaths.cs

```diff
@@ -7,32 +12,49 @@ namespace Microsoft.Extensions.DependencyModel
     internal class DependencyContextPaths
     {
         private static readonly string DepsFilesProperty = "APP_CONTEXT_DEPS_FILES";
```

Comment:
Why don't we use only `APP_CONTEXT_DEPS_FILES`? I was thinking maybe it worth it to return just `string[]` from this class, it would make sure we are loading `.desp` files in the same order that muxer does.